solidity/contracts/GravityERC721.sol

```diff
@@ -0,0 +1,64 @@
+//SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "./Gravity.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import { ERC721Holder } from "@openzeppelin/contracts/token/ERC721/utils/ERC721Holder.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "hardhat/console.sol";
+
+contract GravityERC721 is ERC721Holder, ReentrancyGuard, Ownable{
+	
+	uint256 public state_lastERC721EventNonce = 1;
+	address public state_gravitySolAddress;
+	uint256 constant constant_powerThreshold = 2863311530;
```

Comment:
if we're using arbitrary logic this should not be needed, since we're not validating signatures in this context.

Reply:
Deleting...